README-minishift.md

```diff
@@ -188,17 +188,31 @@ oc create clusterrolebinding federation-admin \
     --serviceaccount="federation-system:default"
 ~~~
 
-Deploy the [cluster registry](https://github.com/kubernetes/cluster-registry) and the namespace where clusters are registered
-(`kube-multicluster-public`):
+Clone the Federation V2 repo and checkout the tag 0.0.4:
 
 ~~~sh
-oc create -f cluster-registry.yaml
+cd federation-v2/
+git checkout tags/v0.0.4
+~~~
+
+Create the required namespaces:
+
+~~~sh
+oc create ns federation-system
```

Comment:
As this is now done explicitly, we can remove line 220 where it says that the `oc apply` created the `federation-system` namespace